From 66ea2c9c4043736676045fa5558db74fa9c48736 Mon Sep 17 00:00:00 2001
From: Brian Wood <brian.j.wood@intel.com>
Date: Thu, 22 Jan 2015 15:11:16 -0800
Subject: [PATCH] REVERT_ME: SDIO quick fix enablement for BCMDHD

This is a quick fix patch to allow for building BCMDHD
driver that supports SDIO on the Cherrytrail T4 platform.

Signed-off-by: Brian Wood <brian.j.wood@intel.com>
---
 drivers/net/wireless/bcmdhd/Makefile | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/drivers/net/wireless/bcmdhd/Makefile b/drivers/net/wireless/bcmdhd/Makefile
index 8470a6f3c5ae..bf0beb4abb46 100644
--- a/drivers/net/wireless/bcmdhd/Makefile
+++ b/drivers/net/wireless/bcmdhd/Makefile
@@ -149,7 +149,10 @@ ifneq ($(CONFIG_BCMDHD_SDIO),)
 # bcn_timeout
   DHDCFLAGS += -DCUSTOM_BCN_TIMEOUT_SETTING=5
 # debug info
-  DHDCFLAGS += -DDHD_WAKE_STATUS
+######## Needs code fix as this includes <linux/wakeup_reason.h> and calls removed
+######## function "check_wakeup_reason()". Commented out for now since its just for
+######## debugging purposes.
+#  DHDCFLAGS += -DDHD_WAKE_STATUS 
 
   DHDCFLAGS += -DWLFC_STATE_PREALLOC
 endif
-- 
2.0.3

